ci-operator/step-registry/rosa/test/e2e/rosa-test-e2e-commands.sh

```diff
@@ -61,10 +59,9 @@ junit_xml="${TEST_PROFILE}-$(date +%m%d%s).xml"
 rosatest --ginkgo.v --ginkgo.no-color \
   --ginkgo.timeout ${TEST_TIMEOUT} \
   --ginkgo.junit-report "${ARTIFACT_DIR}/$junit_xml" \
-  ${LABEL_FILTER_SWITCH} \
-  | sed "s/$AWS_ACCOUNT_ID/$AWS_ACCOUNT_ID_MASK/g" || true
```

Comment:
why did you remove the Mask ?

Reply:
Now it is been masked here: https://github.com/openshift/rosa/blob/master/tests/utils/log/constants.go#L23-L24